<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JKStockTradeDetail</class>
 <widget class="QDialog" name="JKStockTradeDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="lblBuyPrice">
     <property name="text">
      <string>买入价钱：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblBuyCount">
     <property name="text">
      <string>买入数量：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblSellPrice">
     <property name="text">
      <string>卖出价钱：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblCommission">
     <property name="text">
      <string>佣金税费：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblTransfer">
     <property name="text">
      <string>过户税费：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblStampTax">
     <property name="text">
      <string>印花税费：</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
